android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt

```diff
@@ -26,7 +26,7 @@ class HMSVideoView(
         hmsVideoView = view.findViewById(R.id.hmsVideoView)
         hmsVideoView.setEnableHardwareScaler(false)
         hmsVideoView.setMirror(setMirror)
-        hmsVideoView.setAutoSimulcast(isAutoSimulcast)
+        hmsVideoView.disableAutoSimulcastLayerSelect(!isAutoSimulcast)
```

Comment:
@Decoder07 `isAutoSimulcast` is to be renamed to `disableAutoSimulcastLayerSelect`